check/build-changed-protos

```diff
@@ -94,6 +94,17 @@ do
     (set -x; bazel build "${base}_proto"; \
     bazel build "${base}_py_proto"; \
     bazel build "${base}_cc_proto")
+
+    python -m grpc_tools.protoc -I=. --python_out=. --mypy_out=. ${base}.proto
 done
 
+untracked=$(git status --porcelain 2>/dev/null| grep "^?? cirq/google" | cut -d " " -f 2)
```

Comment:
We should either make this (and the loop below) work with files that may include space in the name or add a comment saying we don't support this.

Reply:
I added the comment!